src/TagField.php

```diff
@@ -390,6 +390,9 @@ protected function getOrCreateTag($term)
             $record = Injector::inst()->create($dataClass);
             $record->{$titleField} = $term;
             $record->write();
+            if ($source instanceof SS_List) {
+                $source->add($record);
+            }
```

Comment:
Would you mind explaining what this change does? As far as I can tell it's not used, since `$record` is what is returned

Reply:
This adds the new tag created to the Many_Many / Has_many list. So it completes the relation.

Reply:
Oh of course, because `->add()` internally writes the `$record`